README.md

```diff
@@ -0,0 +1,210 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="app/assets/images/logo-no-background.png" alt="logo" width = "100"/>
+  <h1><b> 💰 Budget App 💸 </b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [💸 Budget APP ](#-budget-app-)
+- [🛠 Built With ](#-built-with-)
+  - [📌 Tech Stack ](#-tech-stack-)
+- [🎲 Key Features ](#-key-features-)
+- [🚀 Live Demo ](#-live-demo-)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 💸 Budget APP <a name="about-project"></a>
+
+**Budget APP** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+# 🛠 Built With <a name="built-with"></a>
+
+## 📌 Tech Stack <a name="tech-stack"></a>
+
+| Domain | Tech Stack |
+|--------|--------|
+| Server | <a href="https://rubyonrails.org/">Ruby on Rails</a> |
+| Database | <a href="https://www.postgresql.org/">PostgreSQL</a> |
+| Testing Framework | <a href="https://github.com/rspec/rspec-rails">RSpec Rails,</a> <a href="https://rubydoc.info/github/teamcapybara/capybara">Capybara</a> |
+| Authentification | <a href="https://github.com/heartcombo/devise#getting-started">Devise</a> |
+| Authorization | <a href="https://github.com/CanCanCommunity/cancancan#installation">cancancan</a> | 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- Features -->
+
+# 🎲 Key Features <a name="key-features"></a>
+
+- Create a new account
+- Log in to an existing account
+- Sign out from a session
+- Create a category
+- Add transaction to a category
+- Check the list of transactions for a particular category
+- Display the total amount of transactions for a particular category
+- Tests for category
+- Tests for transaction
+- Toggle menu
+- Dynamic navbar
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+👉 [Live Demo](https://obed.onrender.com/)<br>
```

Comment:
- [ ] Great job working on this capstone project and updating the ReadMe. Unfortunately, when I click on the [live demo link](https://obed.onrender.com/), it takes like for ever to load and finally gives me a bad gateway page Kindly review your deployment and ensure that the link is working properly.
- **Forever loading page**
![image](https://github.com/obed300/budget-app/assets/62849872/c0db2e74-01b7-44d3-9bf1-c445bae726e8)

- **Bad gateway page**
![image](https://github.com/obed300/budget-app/assets/62849872/c0b4ab82-3fba-4a89-a426-3f10c46f335c)